chia/plotting/check_plots.py

```diff
@@ -201,12 +217,17 @@ def process_plot(plot_path: Path, plot_info: PlotInfo, num_start: int, num_end:
                     caught_exception = True
                     break
 
-                for index, quality_str in enumerate(qualities):
+                for index, proof in enumerate(proofs):
                     # Other plot errors cause get_full_proof or validate_proof to throw an AssertionError
                     try:
                         proof_start_time = round(monotonic() * 1000)
-                        # TODO : todo_v2_plots handle v2 plots
-                        proof = pr.get_full_proof(challenge, index, parallel_read)
+                        if version == PlotVersion.V1:
+                            quality_str = bytes32(proof)
+                            full_proof = pr.get_full_proof(challenge, index, parallel_read)
+                        else:
+                            quality_str = quality_for_partial_proof(proof, challenge)
+                            full_proof = solve_proof(proof)
```

Comment:
what if your plots are not on a machine with a GPU ?

Reply:
I suspect that in that case this call may take a long time, at least for k-32. But if the plot is k-28 I don't expect it to take that long. That target timing is that k-28 can be solved in 8 seconds on a raspberry PI

Reply:
maybe the check plots for v2 does not need include the solver, and we would just do a check for the solver speed separately ?

Reply:
i think we might need to revisit this and make sure we dont need to add some checks for plot_size/gpu so we dont run checks that just hang

Reply:
yes, I expect we'll use the solver service in the future